src/react-apps/applications/service-development/src/features/handleMergeConflict/components/HandleMergeConflictFileList.tsx

```diff
@@ -92,54 +92,57 @@ export class HandleMergeConflictFileList extends
               root: classNames(classes.root),
             }}
           >
-            {repoStatus.contentStatus.length > 0 ?
+            {repoStatus.contentStatus ?
+              repoStatus.contentStatus.length > 0 ?
```

Comment:
this can be a oneliner to avoid the :null :null at the end
repoStatus.contentStatus && repoStatus.contentStatus.length > 0 ?